tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -380,18 +380,13 @@ def test_should_respond_200_task_instance_with_sla_and_rendered(self, session):
     def test_should_respond_200_mapped_task_instance_with_rtif(self, session):
         """Verify we don't duplicate rows through join to RTIF"""
         tis = self.create_task_instances(session)
-        session.query()
-        ti = tis[0]
-        ti.map_index = 1
```

Comment:
no i simplified

Reply:
the issue that forced the simplification...
what the code was doing is this...

> let's take existing ti, notes and all, and change the map index

but now that notes is diff table, you can't just change the map index of ti because that won't also change the index of the keyed obj, namely notes.

simpler to just create _new_ ti.